podcasts/SwitchCell.swift

```diff
@@ -46,4 +46,29 @@ class SwitchCell: ThemeableCell {
 
     override func setSelected(_ selected: Bool, animated: Bool) {}
     override func setHighlighted(_ highlighted: Bool, animated: Bool) {}
+
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        setupAccessibility()
```

Comment:
I did try calling this method from "awakeFromNib" but it was too early as "cellSwitch" and "cellLabels" had values defined in xib file which is not desirable.